### ISSUE TITLE
Possible to autocapture while searching

### ISSUE THREAD

**Issue by ulisesguerrero** [Anveio/mturk-engine#48]
Hello, 

I like the concept of watchers, but have you ever considered using auto-catch or auto-collect within the same seach window (instead of adding watchers)?

Something like this:
1) Search for whatever conditions of reward, To rating, keywords, etc
2) Continuously autocollect top 10 jobs based the search results (no need to add to watchers)
3) Worker simply works and returns jobs he can't execute

In this way, the worker is always engaged in working rather than hunting for jobs or having to know which requesters are currently employing.

Thanks
Ulises

**Comment by Sirensatdawn** [comment]
I agree and have seen a similar script being used where the watcher would auto catch hits based on TO reviews and based of reward amount of the hit.

**Comment by ulisesguerrero** [comment]
Siresatdawn, I have not seen any script that does that, I'm curious, what is the name?

**Comment by xuxoide** [comment]
Would like to see this feature. I work on P9R transcription hits and I have to accept each hit individually. I think each hit have some unique ID so the watcher option wont work for me. I need this script to search and add accept the hits automatically. Thanks!!
